File: src/main/java/com/github/tuk-cps/ExprParser/ExprTreeExp.java
package ExprParser;

import static jAADD.AADD.REAL;

/**
 * Predefined functions: exp
 */
class ExprTreeExp extends ExprTreeFunction {
    ExprTreeExp() { super(REAL, "exp", new ExprTree("x", REAL)); }
    void evalThis() throws ExprError {
        result = getAADDParam(0).exp();
    }
}

File: src/main/kotlin/com/github/tuk-cps/exprParser/ExprTreeExp.kt
package exprParser

import jAADD.AADD

/**
 * Predefined functions: exp
 */
internal class ExprTreeExp : ExprTreeFunction(AADD.Reals, "exp", ExprTree(AADD.Reals, "x")) {
    @Throws(ExprError::class)
    override fun evalUp() {
        value = getAADDParam(0).exp()
    }
}